			     BASH PATCH REPORT
			     =================

Bash-Release:	4.2
Patch-ID:	bash42-020

Bug-Reported-by:	Vincent Sheffer <vince.sheffer@apisphere.com>
Bug-Reference-ID:	<F13C1C4F-C44C-4071-BFED-4BB6D13CF92F@apisphere.com>
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-readline/2011-08/msg00000.html

Bug-Description:

The shared object helper script needs to be updated for Mac OS X 10.7
(Lion, darwin11).

Patch (apply with `patch -p0'):

*** ../bash-4.2-patched/support/shobj-conf	2009-10-28 09:20:21.000000000 -0400
--- support/shobj-conf	2011-08-27 13:25:23.000000000 -0400
***************
*** 158,162 ****
  
  # Darwin/MacOS X
! darwin[89]*|darwin10*)
  	SHOBJ_STATUS=supported
  	SHLIB_STATUS=supported
--- 172,176 ----
  
  # Darwin/MacOS X
! darwin[89]*|darwin1[012]*)
  	SHOBJ_STATUS=supported
  	SHLIB_STATUS=supported
***************
*** 187,191 ****
  
  	case "${host_os}" in
! 	darwin[789]*|darwin10*)	SHOBJ_LDFLAGS=''
  			SHLIB_XLDFLAGS='-dynamiclib -arch_only `/usr/bin/arch` -install_name $(libdir)/$@ -current_version $(SHLIB_MAJOR)$(SHLIB_MINOR) -compatibility_version $(SHLIB_MAJOR) -v'
  			;;
--- 201,205 ----
  
  	case "${host_os}" in
! 	darwin[789]*|darwin1[012]*)	SHOBJ_LDFLAGS=''
  			SHLIB_XLDFLAGS='-dynamiclib -arch_only `/usr/bin/arch` -install_name $(libdir)/$@ -current_version $(SHLIB_MAJOR)$(SHLIB_MINOR) -compatibility_version $(SHLIB_MAJOR) -v'
  			;;
*** ../bash-4.2-patched/patchlevel.h	Sat Jun 12 20:14:48 2010
--- patchlevel.h	Thu Feb 24 21:41:34 2011
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 19
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 20
  
  #endif /* _PATCHLEVEL_H_ */
